templates.dist/fields.php

```diff
@@ -224,4 +224,93 @@ function renderRegionalAddition(string $inputName, array $selectedValue, bool $m
 		$output .= '</select>';
 		return $output;
 	}
-}
\ No newline at end of file
+}
+
+if (!function_exists('renderParkingLot')) {
+	function renderParkingLot(array $parkingArray, string $language, string $locale = 'de_DE'): array
+	{
+		$messages = [];
+		foreach ($parkingArray as $key => $parking) {
+			if (!$parking['Count']) {
+				continue;
+			}
+			$element = sprintf(__('%1$s at %2$s'), getParkingName($key, $parking['Count']), formatPrice($parking['Price'], $language, $locale));
+			if (!empty($parking['MarketingType'])) {
+				$element .= ' (' . $parking['MarketingType'] . ')';
+			}
+			array_push($messages, $element);
+		}
+		return $messages;
+	}
+}
+
+if (!function_exists('formatPrice')) {
```

Comment:
this is global namespace, maybe use a less generic name or put the functions into a namespace? Just a thought...